### ISSUE TITLE
duration returns null on videos over 1 hour long 

### ISSUE THREAD

**Issue by SteeledSlagle13** [VincentJonathan/scrape-yt#7]
Doing a video search with the name on a video that is over 1 hour puts null in the duration of the video.

__Code:__
```JS
const scrapeYt = require("scrape-yt").scrapeYt;

scrapeYt.search("1 hour Trance Mix With Mister Anderson - Transmission 006 [HD]").then(videos => {
  console.log(videos[0]);
});
```

__Console output:__
```js
{
  id: 'L_LUpnjgPso',
  title: 'Fireplace 10 hours full HD',
  duration: null,
  thumbnail: 'https://i.ytimg.com/vi/L_LUpnjgPso/hqdefault.jpg?sqp=-oaymwEjCPYBEIoBSFryq4qpAxUIARUAAAAAGAElAADIQj0AgKJDeAE=&rs=AOn4CLAgH7jbYsBHthSpHJIQlHT5fOCnkA',
  channel: {
    id: 'UCR5hpffFzzlEJoAqY2xotcg',
    name: 'Fireplace 10 hours',
    url: 'https://www.youtube.com/channel/UCR5hpffFzzlEJoAqY2xotcg'
  },
  uploadDate: '3 years ago',
  viewCount: 21224450
}
```

**Comment by SteeledSlagle13** [comment]
possibly due to
[Videos Google API](https://developers.google.com/youtube/v3/docs/videos#resource)
![image](https://user-images.githubusercontent.com/25378777/82108847-4829aa00-96f7-11ea-9e17-f5b58b78e3f0.png)

**Comment by SuspiciousLookingOwl** [comment]
Not able to reproduce
![image](https://user-images.githubusercontent.com/32597776/82112389-9a5de000-9776-11ea-8d48-b2dfd8521ba6.png)
Can you reproduce it consistently?

Video Google API shouldn't matter since I'm scraping from the web.

**Comment by SteeledSlagle13** [comment]
Hey yeah I’m able to reproduce. It’s a consistent issue. I tried in on runkit same issue

![27354F25-E8C9-496B-9451-1D7465D56760](https://user-images.githubusercontent.com/25378777/82112570-9ea4e180-9713-11ea-9cec-9d700b9fc53f.png)

**Comment by SuspiciousLookingOwl** [comment]
Hmm that's weird, how's your Youtube timestamp format? is it like this?
![image](https://user-images.githubusercontent.com/32597776/82112656-0c372900-9779-11ea-9ce8-5e009bba50ef.png)

**Comment by SteeledSlagle13** [comment]
![091A71B0-BAC2-4302-963A-0DBAD0C7D156](https://user-images.githubusercontent.com/25378777/82112680-d52f2c00-9714-11ea-94bf-7743b0450312.png)
![A50AD108-C5B5-4A15-83C7-9D6B2EDB97C2](https://user-images.githubusercontent.com/25378777/82112686-db250d00-9714-11ea-8c8f-3fabc66dcfd4.png)

Looks the same. I’m also on mobile rn. But the issue is the same on my desktop. I ran npm install 2-3 days ago. And the [npm runkit](https://npm.runkit.com/scrape-yt) does the samething my desktop does

**Comment by SuspiciousLookingOwl** [comment]
Seems like regional issue, i connected to USA VPN and tried again, and it gives me `null` on the duration.
Fixing this.

**Comment by SteeledSlagle13** [comment]
Dang:/ thats interesting

**Comment by SuspiciousLookingOwl** [comment]
Ahh, i feel so stupid
I have a function to convert `h:m:s` format to `h.m.s` format, but because i did the regex wrong, it only change the first `:`  so it become `h.m:s`.

Should be fixed now, upgrade to version `1.2.1`

**Comment by SteeledSlagle13** [comment]
Awesome will upgrade!
